pkg/config/get_test.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetValueFromConfig(t *testing.T) {
+	tests := []struct {
+		name           string
+		stringPath     string
+		config         Config
+		expectedOutput interface{}
+	}{
+		{
+			"simple path select string",
+			"cluster.name",
+			Config{
+				Cluster:  Cluster{Name: "clustername", PrivateKey: "privatekey"},
+				Machines: []MachineReplicas{MachineReplicas{Count: 3, Spec: Machine{}}},
+			},
+			"clustername",
+		},
+		{
+			"array path select global",
+			"machines[0].spec",
+			Config{
```

Comment:
You could use a variable to not have to repeat the whole config twice.

Reply:
Done :)